.github/workflows/platformDeploy.yml

```
@@ -157,6 +157,13 @@ jobs:
           ruby-version: '2.7'
           bundler-cache: true
 
+      - uses: actions/cache@v2
```

Let's use v3, because why not?

```suggestion
      - uses: actions/cache@v3
```